src/asset_folder_importer/asset_folder_vsingester/importer_thread.py

```diff
@@ -279,6 +280,13 @@ def run(self):
                 self.db.insert_sysparam("warning", msgstring)
             except FileOnIgnoreList:
                 self.ignored += 1
+            except SweeperHTTPError as e:
+                msgstring = "WARNING: HTTP error communicating with Portal attempting to import %s: %s" % (
+                    filepath, e)
+                self.logger.warning(msgstring)
+                self.logger.warning(traceback.format_exc())
+                self.db.insert_sysparam("warning", msgstring)
+                self.portal_error = True
```

Comment:
this should be logged as an Error instead of a Warning, and you should break the loop at the bottom

Reply:
Done.